### PR TITLE
Fix 🐛 missing VoteableModel export on server & readme

### PR DESCRIPTION
**Pull request by StorytellerCZ**
Also update publish-composite.

### CODE REVIEW

File: server/server.js [hunk 1]
```diff
@@ -39,4 +39,5 @@ VotesCollection.after.remove(function afterRemove(userId, vote) {
     userId && collection && collection.update(vote.linkedObjectId, { $inc: { voteScore: -vote.direction } });
 });
 
+export { VoteableModel } from '../common/voteable-model.js';
```
Comment (copleykj) [inline server/server.js:42]: Something doesn't seem right here. We shouldn't have 2 named exports of the same name. I'll have to check into this and see if there's an underlying issue.

Comment (StorytellerCZ) [reply server/server.js:42]: This is only on server. The problem is was that `VoteableModel` has been exported on client, but not on server.

**Comment by copleykj** [comment]
I cherry picked and edited the one commit. Give it a test and make sure it fixes the issues you were seeing.

**Comment by StorytellerCZ** [comment]
All fine, except #2

**Pull request closed without merging**